docs/book/how-to/setting-up-a-project-repository/set-up-repository.md

```diff
@@ -34,7 +34,21 @@ While it doesn't matter how you structure your ZenML project, here is a recommen
 └── run.py
 ```
 
-All ZenML [Project templates](using-project-templates.md#generating-project-from-a-project-template) are modeled around this basic structure.
+All ZenML [Project
+templates](using-project-templates.md#generating-project-from-a-project-template)
+are modeled around this basic structure. The `steps` and `pipelines` folders
+contain the steps and pipelines defined in your project. If your project is
+simpler you can also just keep your steps at the top level of the `steps` folder
+without the need so structure them in subfolders.
+
+{% hint style="info" %}
+It might also make sense to register your repository as a code repository. These
+enable ZenML to keep track of the code version that you use for your pipeline
+runs. Additionally, running a pipeline that is tracked in a registered code
```

Comment:
I think you can link to the docs jayesh wrote here which went into a lot more detail on this